From 31948901f613a8500e710fe13772591e4ab245bc Mon Sep 17 00:00:00 2001
From: tony <tsyrogit@users.noreply.github.com>
Date: Fri, 11 Aug 2023 22:28:15 +0100
Subject: [PATCH] Changed the compiler warning option to stop warning about
 invalid option for c++ files. Occurred when using cmake and gcc version 12.3.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9cf0ef7..78d2cd8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ add_executable(dictgen dict-generate.cpp)
 target_compile_features(dictgen PUBLIC cxx_std_11)
 target_compile_options(dictgen PRIVATE "-O3")
 
-add_compile_options("-Wextra" "-Wdeclaration-after-statement")
+add_compile_options("-Wall" "-Wextra")
 
 find_library(LIBM m)
 
